### PR TITLE
Feature common components

### PR DESCRIPTION
**Pull request by gustavoleite**
Only the addition of components common to every project.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,23 +1,57 @@
...
   render() {
     return (
-      <View style={styles.container}>
-        <Text>Open up App.js to start working on your app!</Text>
-        <Text>Changes you make will automatically reload.</Text>
-        <Text>Shake your phone to open the developer menu.</Text>
+      <View>
+        <Text style={styles.viewStyle}>
+            Meus documentos
+          </Text>
```
Comment (Morbix) [inline App.js]: Fix identation

File: App.js [hunk 2]
```diff
@@ -1,23 +1,57 @@
...
+            <CardSection>
+              <View style={styles.headerContentStyle}>
+                <Text style={styles.headerTextStyle}>Nome do documento</Text>
+                <Text>Número do documento</Text>
+              </View>
+            </CardSection>
+            <CardSection>
+              <Image
+                style={styles.imageStyle}
+              />
```
Comment (Morbix) [inline App.js]: close in the same line of style, because you dont have children

File: App.js [hunk 3]
```diff
@@ -1,23 +1,57 @@
...
 export default class App extends React.Component {
+
   render() {
     return (
-      <View style={styles.container}>
-        <Text>Open up App.js to start working on your app!</Text>
-        <Text>Changes you make will automatically reload.</Text>
-        <Text>Shake your phone to open the developer menu.</Text>
+      <View>
+        <Text style={styles.viewStyle}>
```
Comment (Morbix) [inline App.js]: Why this text have a style called `viewStyle`?

File: src/components/common/Confirm.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+const Confirm = ({ children, visible, onAccept, onDecline }) => {
+  const { containerStyle, textStyle, CardSectionStyle } = styles;
+
+  return (
+    <Modal
+      visible={visible}
+      transparent
+      animationType="slide"
+      onRequestClose={() => {}}
+    >
```
Comment (Morbix) [inline src/components/common/Confirm.js:15]: close with the last attribute

File: App.js (hunk 1 above)
Comment (gustavoleite) [reply App.js]: Fix

File: App.js (hunk 2 above)
Comment (gustavoleite) [reply App.js]: Fix!

File: src/components/common/Confirm.js (hunk 4 above)
Comment (gustavoleite) [reply src/components/common/Confirm.js:15]: The broker asks you to do this, look:
image

File: App.js (hunk 3 above)
Comment (gustavoleite) [reply App.js]: Fix!

File: src/components/common/Confirm.js (hunk 4 above)
Comment (Morbix) [reply src/components/common/Confirm.js:15]: So ok.

**Review by Morbix — APPROVED** [review]

**Pull request merged**